requirements/edx/base.txt

```diff
@@ -115,6 +115,7 @@ edx-completion==0.1.8
 edx-django-oauth2-provider==1.3.4
 edx-django-release-util==0.3.1
 edx-django-sites-extensions==2.3.1
+edx-django-utils==0.3.0   # via edx-drf-extensions
```

Comment:
@douglashall: Do you know why this dependency wasn’t pulled in via the merged PR?

Reply:
Yes, it is because I hand edited the edx-drf-extensions requirement in the merged PR. This PR should set us back on the right track.

Reply:
Got it. Thanks.